packages/uma-auth-client/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@uma-sdk/uma-auth-client",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/uma-auth-client.umd.cjs",
+  "exports": {
+    ".": {
+      "types": "./dist/uma-auth-client.d.ts",
+      "import": {
+        "types": "./dist/uma-auth-client.d.ts",
+        "default": "./dist/uma-auth-client.js"
+      },
+      "require": {
+        "types": "./dist/uma-auth-client.d.ts",
+        "default": "./dist/uma-auth-client.umd.cjs"
+      }
+    }
+  },
+  "module": "./dist/uma-auth-client.js",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
+    "lint:fix": "eslint --fix .",
+    "preview": "vite preview",
+    "format:fix": "prettier src --write",
+    "format": "prettier src --check"
+  },
+  "dependencies": {
```

Comment:
Do we really need all of these deps? I feel like we should really try to minimize what we depend on in an SDK for size purposes. It's also possible some of these should be peer dependencies. I know your next PR is breaking out the webcomponents implementation, so maybe I'm getting ahead of myself here, but we should really think critically about our deps.

Reply:
moved some dependencies to dev dependencies in https://app.graphite.dev/github/pr/uma-universal-money-address/uma-js-sdk/105/Add-button-customizability-wrap-web-component-in-react-component

From what I understand peerDependencies doesn't actually mean a single version of the package is used; however, I set react and react-dom to external in the vite config so they are only bundled once reducing the size (I checked in the bundle output in both cases and this appears to be the case)